### ISSUE TITLE
[BUG] Public transport departure Pie Chart at Freiburg Hauptbahnhof faulty?

### ISSUE THREAD

**Issue by UJehle** [goat-community/goat#2101]
### Summary 📃

1. When I click on the pie chart at Freiburg Main station, the result window is empty.
2. Furthermore, there are elements in the legend which don't appear in the map.
![grafik](https://user-images.githubusercontent.com/49870195/228760861-bd8efa33-9743-4b7f-bd77-0197e1c6f327.png)

### Expected Behaviour ✔

1. The result window contains a result. 
2. The elements in the legend fit to the elements in the map.

### Current Behaviour 🔦

1. The result window doesn't contain a result. 
2. The elements in the legend don't fit to the elements in the map.

### Steps to reproduce 👣

1. Open GOAT Freiburg
2. Switch on the pie charts
3. Click on the main station
4. See an empty result window

### Possible Implementation 📐

This issue might be similar to the one we had at Munich main station.

**Comment by UJehle** [comment]
Are these changes already online? 
"High speed rail" is now appearing in the pie chart, but "inter regio" is missing although there should be several trains departing. 
![grafik](https://user-images.githubusercontent.com/49870195/233838105-09cc8dce-9041-459c-b63a-c47753a1e050.png)

Also "S-Bahn" is missing in our indicator. 
![grafik](https://user-images.githubusercontent.com/49870195/233838297-ea3dfa92-d882-4914-b7a2-baaf70d4538e.png)


Btw. what is the difference between "Fernverkehr" and "Hochgeschwindigkeitszug"?

**Comment by Ebubeker** [comment]
I think the missing ones have to do with the data, since these are all the ones which have been provided. The S-bahn might also be included in the **Rail** since with what I saw online it is "The S-Bahn is the name of hybrid urban-suburban _rail_ systems"

As for Fernverkehr and Hochgeschwindigkeitszug maybe they are different in the aspect of speed and station stops, but I am not quite sure since I am not of this field.

**Comment by Ebubeker** [comment]
"Fernverkehr" and "Hochgeschwindigkeitszug" are both German words related to transportation, specifically rail transport. However, they refer to different concepts.

"Fernverkehr" is a German term that refers to long-distance travel, usually by train. It can be translated into English as "long-distance traffic" or "long-distance transportation." This term includes all types of long-distance train services, including regional express trains, intercity trains, and high-speed trains.

On the other hand, "Hochgeschwindigkeitszug" is a German term that specifically refers to high-speed trains, also known as bullet trains. This term can be translated into English as "high-speed train." Hochgeschwindigkeitszüge are capable of travelling at speeds of 250 km/h or higher, and they typically connect major cities and destinations within a country or across borders.

In summary, while Fernverkehr refers to all long-distance train services, including high-speed trains, Hochgeschwindigkeitszug specifically refers to high-speed trains.

This is what chat gpt said about this

**Comment by UJehle** [comment]
Okay I see, I thought you might have discussed something that was not documented in the issue regarding the modes, as now there were even more modes than before and the numbers in the pie charts seemed to be correct at Freiburg Hauptbahnhof. Thus, I thought it's on purpose and wasn't aware that this is a data issue. 

The category "Bahn" is currently not included, so this would be new?
